site/src/components/Alert/ErrorAlert.tsx

```diff
@@ -1,28 +1,55 @@
 import AlertTitle from "@mui/material/AlertTitle";
-import { getErrorDetail, getErrorMessage } from "api/errors";
+import { getErrorDetail, getErrorMessage, getErrorStatus } from "api/errors";
 import type { FC } from "react";
+import { Link } from "../Link/Link";
 import { Alert, AlertDetail, type AlertProps } from "./Alert";
 
 export const ErrorAlert: FC<
 	Omit<AlertProps, "severity" | "children"> & { error: unknown }
 > = ({ error, ...alertProps }) => {
 	const message = getErrorMessage(error, "Something went wrong.");
 	const detail = getErrorDetail(error);
+	const status = getErrorStatus(error);
 
 	// For some reason, the message and detail can be the same on the BE, but does
 	// not make sense in the FE to showing them duplicated
 	const shouldDisplayDetail = message !== detail;
 
-	return (
-		<Alert severity="error" {...alertProps}>
-			{detail ? (
+	const body = () => {
+		// When the error is a Forbidden response we include a link for the user to
+		// go back to a known viewable page.
+		// Additionally since the error messages and details from the server can be
+		// missing or confusing for an end user we render a friendlier message
+		// regardless of the response from the server.
+		if (status === 403) {
+			return (
+				<>
+					<AlertTitle>{message}</AlertTitle>
+					<AlertDetail>
+						{detail}{" "}
```

Comment:
I know theoretically we're now always setting `Detail` for 403 responses, but what if it comes back undefined from some endpoint that doesn't use `httpapi.Forbidden` when it should? do we care about handling that? maybe we don't and it's fine 🤷‍♀️